src/main/java/io/vertx/ext/mail/MailAttachment.java

```diff
@@ -74,6 +75,41 @@ static MailAttachment create(MailAttachment other) {
   @Fluent
   MailAttachment setData(Buffer data);
 
+  /**
+   * Gets the data stream.
+   *
+   * @return the data stream
+   */
+  ReadStream<Buffer> getStream();
+
+  /**
+   * Sets the data stream.
+   *
+   * @param stream data stream to be used at attachment
+   * @return this to be able to use it fluently
+   */
+  @Fluent
+  MailAttachment setStream(ReadStream<Buffer> stream);
+
+  /**
+   * Gets the size of the attachment.
+   *
+   * @return the size of the attachment
+   */
+  Integer getSize();
+
+  /**
+   * Sets the size of the attachment.
+   *<p>
+   * It is needed when using ReadStream for the MailAttachement.
+   *</p>
+   *
+   * @param size the size of the attachment
+   * @return this to be able to use it fluently
+   */
+  @Fluent
+  MailAttachment setSize(Integer size);
```

Comment:
Same here, why not just `int`?

Reply:
I was thinking to check whether the size if set or not. But looks like using `0` as not-set works well too. :)

Reply:
`-1` would be better I think